### ISSUE TITLE
The "happy bunny license" is nonfree, confusing and unenforceable

### ISSUE THREAD

**Issue by GarethNelson** [g-truc/glm#511]
While funny, the clause that "you make a bunny unhappy" makes the software nonfree, and the military are unlikely to care if they ever choose to make use of the code.

All this license does is complicate matters when mixing code with other licenses. Since it's nonfree, it's also GPL incompatible too.

More confusion: the wording on the project's website uses "or" - as in it's available under the MIT license OR the happy bunny license or as in "contains code under both licenses".

Some clarity (and even better, dropping this silly license) would be much appreciated.

**Comment by evg-zhabotinsky** [comment]
I wanted to create a new issue, but I'll just put it here :smile:

This "bunny" clause, together with "no warranty" one, goes AFTER following one:

The _**above** copyright notice and **this** permission notice_ shall be included in
all copies or substantial portions of the Software.

Not a word about "below", so everything below can be dropped!

I get it why "no warranty" should be optional for redistribution.

_But why the bunny?!!_

**Comment by cgmb** [comment]
I was similarly confused by the manual, and I came across this page when trying to Google the license to learn more about it. To save others the time I spent on this, the dual-licensing was clarified in d8576b94f35cc6d9e590b2a7957b1d44e8f3c4db.

> GLM can be distributed and/or modified under the terms of either a) The Happy Bunny License, or b) the MIT License.

The licenses were later moved into the manual and that introduction was lost in the transition, but I would presume the intention remains the same.
